### PR TITLE
added banner and native ad in applovin

### PR DESCRIPTION
**Pull request by Shubh-appyhigh**

### CODE REVIEW

File: lib/src/applovin/native_ad/applovin_native_ad.dart [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+        return const Size(468, 250);
+      case AdSdkAdSize.smallNative:
+        return const Size(468, 72);
+      default:
+        return const Size(468, 72);
+    }
+  }
+
+  @override
+  Future<void> loadAd({required AdLoadListener adLoadListener}) async {
```
Comment (ukbaranwal31) [inline lib/src/applovin/native_ad/applovin_native_ad.dart:42]: Can't we do the same for banner ads?

File: lib/src/applovin/native_ad/applovin_native_ad.dart [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+  );
+
+  bool _isAdLoaded = false;
+
+  bool _failedToLoad = false;
+
+  final MaxNativeAdViewController _nativeAdViewController =
+      MaxNativeAdViewController();
+
+  MaxNativeAdView? nativeAdViewRef;
```
Comment (ukbaranwal31) [inline lib/src/applovin/native_ad/applovin_native_ad.dart]: make it private, and nativeAd as name is fine

File: lib/src/applovin/native_ad/applovin_native_ad.dart [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+  ApplovinNativeAd(
+    super.adId,
+    this._sdkAdSize,
+  );
+
+  bool _isAdLoaded = false;
+
+  bool _failedToLoad = false;
+
+  final MaxNativeAdViewController _nativeAdViewController =
```
Comment (ukbaranwal31) [inline lib/src/applovin/native_ad/applovin_native_ad.dart]: keep the name as controller

File: lib/src/applovin/native_ad/applovin_native_ad.dart (hunk 1 above)
Comment (Shubh-appyhigh) [reply lib/src/applovin/native_ad/applovin_native_ad.dart:42]: No, there are no controller argument or or any function through which we can load the widget banner ad

File: lib/src/applovin/native_ad/applovin_native_ad.dart (hunk 3 above)
Comment (Shubh-appyhigh) [reply lib/src/applovin/native_ad/applovin_native_ad.dart]: done

File: lib/src/applovin/native_ad/applovin_native_ad.dart (hunk 2 above)
Comment (Shubh-appyhigh) [reply lib/src/applovin/native_ad/applovin_native_ad.dart]: done

**Pull request closed without merging**